pages/marker.html

```diff
@@ -8,13 +8,64 @@
 
     <!-- import the webpage's stylesheet -->
     <link rel="stylesheet" href="../style.css" />
+    <link rel="stylesheet" href="../components/css/main.css" />
   </head>
   <body>
     <div class="header">
       <a href="../index.html">
-        <img src="img/logo.png" alt="logo" />
+        <img src="../assets/img/logo.png" alt="logo" />
         <p>AR.js Studio</p>
       </a>
     </div>
+
+    <div class="pages-content">
+      <a class="primary-button" href="../index.html">←</a>
```

Comment:
Should we implement buttons as regular HTML buttons? Maybe it's ok for now but I guess we need this change in the future, what do you think?

Reply:
Also, I added the arrow as a single character, but I guess an icon would be better, what do you think?

Reply:
I'm not sure about the anchor vs HTML button, but there shouldn't be a href attribute for these buttons since they aren't used for routing. Right now, it resets the page because it has an empty href attribute. Later when we actually implement the upload function, we would use onClick attribute to invoke the function.

Reply:
Good catch! Removed href from upload buttons 😃